README.adoc

```diff
@@ -0,0 +1,106 @@
+= PartiQL Language Specification
+
+This is the LaTeX source for the [PartiQL] specification.
+
+== How to Build the Spec
+
+=== Pre-requisites
+
+1. Install link:https://github.com/rbenv/rbenv#installation[rbenv]
++
+[source, shell]
+.terminal
+```
+brew install rbenv
+rbenv init
+```
+
+2. Install the latest version of ruby
++
+[source, shell]
+.terminal
+```
+rbenv install $(rbenv install -l | grep -v - | tail -1)
+```
+
+3. Install rbenv shell integration (replace .zshrc with .bash_profile if you use bash)
++
+[source,shell]
+.terminal
+```
+echo "\n# rbenv integration" >> ~/.zshrc
+echo "eval \"\$(rbenv init -)\"" >> ~/.zshrc
+source ~/.zshrc
+```
+
+4. Install Vollkorn & Iosevka fonts (Open Font License)
++
+[source,shell]
+.terminal
+```
+mkdir fonts
+curl -L -o fonts/vollkorn.zip http://vollkorn-typeface.com/download/vollkorn-4-105.zip
+unzip fonts/vollkorn.zip -d fonts/vollkorn
+curl -L -o fonts/iosevka.zip https://github.com/be5invis/Iosevka/releases/download/v16.0.2/ttf-iosevka-term-slab-16.0.2.zip
+unzip fonts/iosevka.zip -d fonts/iosevka
+
+```
+
+5. Install fonts and such needed by AsciiMath
++
+[source,shell]
+.terminal
+```
+brew install glib gdk-pixbuf cairo pango cmake font-computer-modern
```

Comment:
`font-computer-modern` wasn't available to homebrew by default. To install `font-computer-modern`, I had to first install `homebrew/cask-fonts` ([instructions here](https://github.com/Homebrew/homebrew-cask-fonts)). Could leave a comment mentioning this extra step.